pandas/core/indexes/base.py

```diff
@@ -799,6 +799,18 @@ def is_(self, other) -> bool:
         See Also
         --------
         Index.identical : Works like ``Index.is_`` but also checks metadata.
+
+        Examples
+        --------
+        >>> idx1 = pd.Index(['1', '2', '3'])
+        >>> idx2 = pd.Index(['1', '2', '3'])
+        >>> idx2.is_(idx1)
+        False
+
+        >>> idx1 = pd.Index(['1', '2', '3'])
+        >>> new_name = idx1
+        >>> new_name.is_(idx1)
+        True
```

Comment:
to make it clearer, maybe we could have an example like
```
>>> idx1.is_(idx1.view())
True
>>> idx1.is_(idx1.copy())
False
```

Reply:
Done